account_bank_statement_import_qif/wizards/account_bank_statement_import_qif.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2015 Odoo S. A.
+# Copyright 2015 Laurent Mignon <laurent.mignon@acsone.eu>
+# Copyright 2015 Ronald Portier <rportier@therp.nl>
+# Copyright 2016-2017 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import dateutil.parser
+
+from odoo.tools.translate import _
+from odoo import api, models
+from odoo.exceptions import UserError
+
+
+class AccountBankStatementImport(models.TransientModel):
+    _inherit = "account.bank.statement.import"
+
+    @api.model
+    def _check_qif(self, data_file):
+        return data_file.strip().startswith(b'!Type:')
+
+    def _parse_file(self, data_file):
```

Comment:
Looking on how the `import_file` is used, `self` here seems to be a record set. Maybe looping on self would be better?